### DISCUSSION TITLE
Issue with Cursor Navigation Using Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by himatech** [freeplane/freeplane#1651] [Help]
Hello Freeplane community,

I hope this message finds you well. I'm currently facing an issue with cursor navigation using keyboard shortcuts in Freeplane, and I'm seeking assistance to resolve it.

Specifically, I've encountered problems when attempting to navigate within the Notes panel. When I use the ⌘+↑ shortcut to move the cursor to the beginning of the text, it doesn't function as expected; instead, it shifts the node upwards. Similarly, when using ⌘+↓ to position the cursor at the end of the text, it shifts the node downward.

Additionally, I've observed that attempting to move the cursor one word at a time using ⌥+← or → does not produce the desired result.

Are there alternative keyboard shortcuts or settings that I may have overlooked, please let me know.

Thank you in advance for your assistance.

**Comment by i-plasm** [comment]
Hello @himatech , I just checked and all these shortcuts work in my mac when working on the notes panel. But the key is the Notes panel MUST be focused first. If you don't focus the notes panel first, then the shortcuts will be applied on the mindmap (or on whatever component has focus), that's why you saw behaviour that moved nodes up/downward. To focus the notes panel just click on its text area (or assign a shortcut to the menu at ` Edit > Note > Switch Note/Map Focus` for easy toggling the focus of the notes panel).
